src/main/groovy/de/itemis/mps/gradle/GitBasedVersioning.groovy

```diff
@@ -24,7 +24,17 @@ class GitBasedVersioning {
      * @throws org.gradle.api.GradleException if the branch name cannot be determined
      */
     static String getGitBranch() throws GradleException {
-        String gitBranch = getCommandOutput('git rev-parse --abbrev-ref HEAD').replace("/", "-")
+        String gitBranch
+        String gitBranchTC=System.getenv('teamcity_build_branch')
+        
+        if(gitBranchTC!=null && !gitBranchTC.isEmpty()){
+            gitBranch=gitBranchTC.replace("/", "-")
```

Comment:
`replace("/", "-")` is duplicated between the `if` and `else` branch, can you remove the duplication?